src/Chunk.ts

```diff
@@ -395,17 +409,31 @@ export default class Chunk {
 		const exposedVariables = new Set<Variable>(
 			this.dynamicEntryModules.map(({ namespace }) => namespace)
 		);
+
+		// Temporary set to check for conflicts
+		const knownExportNames = new Set<string>();
+		for (const name of this.exportsByName.keys()) {
+			knownExportNames.add(name);
+		}
```

Comment:
Question (probably the only one): Why do we need to generate `knownExportNames`? Couldn't we just write everywhere instead of `knownExportNames.has(...)` `this.exportsByName.has(...)`? Then we can avoid the need to keep the Set in sync with the Map.

Reply:
It's how I could keep the logic in my head :sweat_smile:.

But indeed you're right, it is not needed, great! I took it out.